### DISCUSSION TITLE
How to embed this into a Javascript site?

### DISCUSSION THREAD

**Discussion by somguyth** [highfestiva/finplot#220] [General]
I'm building a site (React.JS front-end with Python backend). Would there be a way to embed these plots into my site somehow? I don't mean pre-rendering a PNG in a headless fashion and then displaying it, I mean displaying the actual interactive plot that people can zoom in on etc.

**Comment by highfestiva** [comment]
Nope, sorry. Perhaps [Bokeh](https://github.com/highfestiva/coinplot) is a start?

**Comment by noclipper** [comment]
Is this still the case? It is not possible to embed a plot into a div or html?

**Comment by highfestiva** [comment]
Still the case, based on PyQt, and not going to replace the backend.
